modules/promotions/admin-menu-items/custom-icons-promotion-item.php

```diff
@@ -21,14 +20,14 @@ public function get_page_title() {
 	}
 
 	public function get_promotion_title() {
-		echo sprintf(
+		return sprintf(
 			/* translators: %s: br  */
 			esc_html( 'Enjoy creative freedom %s with Custom Icons', 'elementor' ),
 			'<br />'
 		);
 	}
 
-	public function set_list() {
+	public function get_content_lines(): array {
```

Comment:
If we are already changing the API, why not to change the access modifiers as well?